src/components/change-view.jsx

```diff
@@ -193,7 +199,6 @@ export default class ChangeView extends React.Component {
             <i className="fa fa-pencil" aria-hidden="true" />
             <a className="lnk-action" href="#" onClick={this._annotateChange}>Update Record</a>
             <i className="fa fa-list" aria-hidden="true" />
-            <Link to={`/${this.props.pageFilter}`} className="lnk-action">Back to list view</Link>
```

Comment:
The `<i className="fa fa-list">` above this needs to come out, too, right?

Reply:
D'oh! Of course, removed.